requirements/edx/github.txt

```diff
@@ -78,7 +78,7 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.4#egg=XBlock==0.4.4
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@0.2.6#egg=ora2==0.2.6
+-e git+https://github.com/edx/edx-ora2.git@0.2.7#egg=ora2==0.2.7
```

Comment:
What does it mean to remove the "-e"? Why was it there before, and why do we no longer need it?